dev-docs/source/Testing/Utility/FilterEventsInterfaceTest.rst

```diff
@@ -0,0 +1,60 @@
+.. _filter_event_testing:
+
+Filter Events Interface Testing
+===============================
+
+.. contents::
+   :local:
+
+*Prerequisites*
+
+- Download the `Usage Examples <http://download.mantidproject.org>`_
+
+*Note: At the time of writing this test, there's an issue pending with data validation on this interface(#36073); if this issue hasn't been
+corrected yet, please remind the team about it, else update documentation and remove this note.*
+
+Filter Events
+--------------
+
+
+**Time required: 10-15 minutes**
+
+--------------
+
+#. Open ``Interfaces`` > ``Utility`` > ``Filter Events``
+#. Browse for file 'CNCS_7860_Event.nxs' from the Usage Data set.
+#. Click ``Load``. After few seconds, plot should be updated with a graph of summed up Counts vs.Time.
+#. ``_Summed_..`` workspace should be generated on the ADS, as well as ``CNCS_7860_Event`` workspace.
+#. Clicking on ``Refresh`` button should update the drop-down list with the name of the loaded workspace.
+#. Check Vertical Range Sliders work properly. You can't overlap the sliders or cross the sliders.
+   Additionally, placing lower slider at the bottom of the range allows the other slider to move freely on the whole range.
```

Comment:
yeah, there is a little bit of padding. I think is coded on the sliders but I'm not sure. I'll investigate it tomorrow. 
Anyway I will rephrase it to avoid confusion with testers.